docs/source/install/rhel6.rst

```diff
@@ -62,8 +75,8 @@ Install MongoDB, RabbitMQ, and PostgreSQL.
     sudo chkconfig mongod on
     sudo chkconfig rabbitmq-server on
 
-    # Install and configure postgres 9.4
-    sudo yum -y localinstall http://yum.postgresql.org/9.4/redhat/rhel-6-x86_64/pgdg-centos94-9.4-1.noarch.rpm
+    # Install and configure postgres 9.4. Based on OS type install the ``redhat`` one or ``centos`` one.
+    sudo yum -y localinstall http://yum.postgresql.org/9.4/redhat/rhel-6-x86_64/pgdg-${os}-94-9.4-1.noarch.rpm
```

Comment:
That was my original commit but then since this is in bash code block, it didn't stand out well. Anyways, see this https://www.evernote.com/l/ANEvhqOkHPlN-5ERcQwfpNdbI7MaCMAvv9AB/image.png

Now if they copy paste both the lines without reading, we'll be fine because they are `if` wrapped. This is safer I think - 78bb4ac